### PR TITLE
Added WPS Download handler

### PR DESCRIPTION
**Pull request by pmbohm**
Works with http://127.0.0.1:8080/aodn-portal/home?uuid=8c5f876e-7394-4641-b2d0-3f6c1664662f&name=TestAnmnTS
Needs `geonetwork.url = "http://catalogue-rc.aodn.org.au/geonetwork"` in config.groovy

### CODE REVIEW

**Comment by jkburges** [comment]
I have broken this PR with [my commit](https://github.com/aodn/aodn-portal/commit/c4d92003886cbcc69bb268e4c9745926ddcee32a).  I will fix it here, since to be fair, this PR was up first!

File: web-app/js/portal/cart/WpsDownloadHandler.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.cart');
+
+Portal.cart.WpsDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
```
Comment (jkburges) [inline web-app/js/portal/cart/WpsDownloadHandler.js]: This class is *very* similar to `GogoduckDownloadHandler`, apart from some small differences in labels, URL construction, and subset parameters.  But overall, the logic is the same.

Consider extracting out the common code (to a `Portal.cart.AsyncDownloadHandler` class, perhaps).

Comment (pmbohm) [inline web-app/js/portal/cart/WpsDownloadHandler.js]: @jkburges Yes now that the smoke has cleared, I agree with extracting the common code but i couldnt finish all tests in time. Feel free to take a look at https://github.com/aodn/aodn-portal/issues/1997 and make any changes to get it in if you wish

Comment (danfruehauf) [inline web-app/js/portal/cart/WpsDownloadHandler.js]: Don't bother too much extracting common code with that and the gogoduck handler - gogoduck handler should die sooner than later.

Comment (pmbohm) [inline web-app/js/portal/cart/WpsDownloadHandler.js]: Well the choices are there now on a separate branches, so take your pick tomorrow with @jkburges . Get healthy!

**Comment by jkburges** [comment]
The test record is in geonetwork-rc, not 123 - I have changed that in the OP.

**Comment by jkburges** [comment]
I don't seem to have a download option which triggers WPS?

**Comment by jkburges** [comment]
> I don't seem to have a download option which triggers WPS?

Take that back - yes I do.

**Pull request closed without merging**